scripts/ChangesSinceLastRelease.txt

```diff
@@ -2,3 +2,854 @@
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CREDENTIAL_ATTESTATION_VERSION_8 added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CREDENTIAL_DETAILS_VERSION_3 added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CREDENTIAL_DETAILS_VERSION_4 added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CREDENTIAL_HINT_CLIENT_DEVICE added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CREDENTIAL_HINT_HYBRID added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CREDENTIAL_HINT_SECURITY_KEY added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CTAP_TRANSPORT_BLE_STRING added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CTAP_TRANSPORT_HYBRID_STRING added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CTAP_TRANSPORT_INTERNAL_STRING added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CTAP_TRANSPORT_NFC_STRING added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CTAP_TRANSPORT_SMART_CARD added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CTAP_TRANSPORT_SMART_CARD_STRING added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_CTAP_TRANSPORT_USB_STRING added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_RP_ENTITY_INFORMATION_VERSION_1 added
+Windows.Win32.Networking.WindowsWebServices.Apis.WEBAUTHN_USER_ENTITY_INFORMATION_VERSION_1 added
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_ADD_AUTHENTICATOR_OPTIONS.cbPluginIdKey removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_ADD_AUTHENTICATOR_OPTIONS.pbPluginIdKey removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_CREDENTIAL_DETAILS_LIST.cbNonce removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_CREDENTIAL_DETAILS_LIST.cbSignature removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_CREDENTIAL_DETAILS_LIST.pbNonce removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_CREDENTIAL_DETAILS_LIST.pbSignature removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_REMOVE_AUTHENTICATOR_OPTIONS removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_UPDATE_AUTHENTICATOR_DETAILS.cbNonce removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_UPDATE_AUTHENTICATOR_DETAILS.cbPluginIdKey removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_UPDATE_AUTHENTICATOR_DETAILS.cbSignature removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_UPDATE_AUTHENTICATOR_DETAILS.pbNonce removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_UPDATE_AUTHENTICATOR_DETAILS.pbPluginIdKey removed
+Windows.Win32.Networking.WindowsWebServices.EXPERIMENTAL_WEBAUTHN_PLUGIN_UPDATE_AUTHENTICATOR_DETAILS.pbSignature removed
```

Comment:
Folks targeting downlevel will need these old constants. We typically don't remove anything during an SDK update. (Surprised to even see this in an SDK minor increment.)

Reply:
See prior; experimental APIs get removed and eventually replaced.